service/src/main/java/com/derpgroup/derpwizard/resource/HomeResource.java

```diff
@@ -37,7 +37,7 @@
  * @since 0.0.1
  */
 @Path("/")
-@Produces(MediaType.APPLICATION_JSON)
+@Produces(MediaType.TEXT_PLAIN)
```

Comment:
It was breaking my Postman tests, because it was returning with Content-Type that didn't match what was coming out.